src/components/question-editor/QuestionEditor.tsx

```diff
@@ -130,63 +156,108 @@ const QuestionEditor = () => {
     </Box>
   ))
 
+  const disabledButton =
+    type === 'openAnswer' ? text && openAnswer : text && answers[0]?.text
+
+  const buttons = (
+    <Box sx={styles.buttons}>
+      <AppButton onClick={() => navigate(-1)} variant={ButtonVariantEnum.Tonal}>
+        {t('common.cancel')}
+      </AppButton>
+      <AppButton
+        disabled={!disabledButton}
+        loading={loading}
+        type={ButtonTypeEnum.Submit}
+      >
+        {t('common.save')}
+      </AppButton>
+    </Box>
+  )
+
+  const optionsMock = ['English', 'Music']
+
   return (
-    <Box sx={styles.root}>
-      <Box sx={styles.questionHeader}>
+    <Box
+      component={ComponentEnum.Form}
+      onSubmit={handleSubmit}
+      sx={styles.root}
+    >
+      <AppTextField
+        InputLabelProps={styles.titleLabel}
+        InputProps={styles.titleInput}
+        fullWidth
+        inputProps={styles.input}
+        label={title ? '' : t('questionPage.untitled')}
```

Comment:
Try to hide label in InputLabelProps styles

Reply:
Fixed